db/models.py

```diff
@@ -1,5 +1,11 @@
+from typing import Any
+
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
 
+import settings
```

Comment:
refactor this place with recommendation
 https://docs.djangoproject.com/en/4.2/topics/settings/#using-settings-in-python-code